tests/test_scrubbers.py

```diff
@@ -38,3 +41,38 @@ def test_faker_scrubber_charfield(self):
 
         self.assertNotEqual(data.last_name, 'Foo')
         self.assertNotEqual(data.last_name, '')
+
+    def test_faker_scrubber_with_provider_arguments(self):
+        """
+        Use this as an example for Faker scrubbers with parameters passed along
+        """
+        data = DataFactory.create(ean8='8')
+        with self.settings(DEBUG=True, SCRUBBER_GLOBAL_SCRUBBERS={'ean8': scrubbers.Faker('ean', length=8)}):
+            call_command('scrub_data')
+        data.refresh_from_db()
+
+        # The EAN Faker will by default emit ean13, so this should fail if the parameter was ignored
+        self.assertEquals(8, len(data.ean8))
+
+        # Add a new scrubber for ean13
+        with self.settings(DEBUG=True, SCRUBBER_GLOBAL_SCRUBBERS={'ean8': scrubbers.Faker('ean', length=13)}):
+            call_command('scrub_data')
+        data.refresh_from_db()
+
+        # make sure it doesn't reuse the ean with length=8 scrubber
+        self.assertEquals(13, len(data.ean8))
+
+    def test_faker_scrubber_datefield(self):
+        """
+        Use this as an example for Scrubber's capability of optimistically Casting to the current field's type
+        There is a bug with django < 2.2 and sqlite, that's why we don't run the test there.
```

Comment:
we're currently not testing django 2.2. Could you please also add that to the PR? Should be just a matter of adding travis and tox configs.

Reply:
Thanks for pointing that out. Acutally it works in django 2.1 as well. Fixed it AND added the django 2.2 test.